recipe/deploy/vendors.php

```diff
@@ -6,6 +6,8 @@
 
 set('composer_options', '--verbose --prefer-dist --no-progress --no-interaction --no-dev --optimize-autoloader');
 
+set('composer_version', null);
```

Comment:
Please add a comment before set() function about what it is, what format, code example, etc.